analysis/analyze-hotspots/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'application'
-    id 'org.openjfx.javafxplugin' version '0.0.5'
+    id 'org.openjfx.javafxplugin' version '0.0.8'
```

Comment:
Could we make a new variable in the version.gradle file to avoid having to update this again in the future?

Something like $javaFxVersion?

Reply:
This is a good shout, but I think it'd require creating a `version.gradle` file for each sample. In the Java samples, each sample is it's own 'gradle project', so we wouldn't have a central place to change this variable from? Might be wrong though

Reply:
Hmm. We only have one on Android which we share across several projects.

Just need to add a line to the gradle file to tell it where the version.gradle can be found with something like:

`apply from: '../version.gradle'` for a situation where the version.gradle file is up one level from the current gradle file location.

Reply:
If I put the JavaFX version in a project level `version.gradle` file, then use a `defaultConfig` block inside the sample's `build.gradle` file, I get an error that the plugins should contain a string literal or a string with property replacement. 

I think the plugin version has to be [string literal](https://docs.gradle.org/current/userguide/plugins.html#sec:constrained_syntax). If we wanted to use a [variable](https://docs.gradle.org/current/userguide/plugins.html#sec:plugin_version_management) we would need to create a `pluginManagement` block in the `settings.gradle` file. I'm not sure how this would work when the samples are individual gradle projects, but I can look into it!

Reply:
Looks like you're on the right track and if the intention is to keep all samples on the same javafx version going forward I reckon it's probably worth the effort